src/main/groovy/jenkins/automation/rest/RestApiScriptRunner.groovy

```diff
@@ -3,7 +3,6 @@
 package jenkins.automation.rest
 
 import javaposse.jobdsl.dsl.DslScriptLoader
-import javaposse.jobdsl.dsl.ScriptRequest
```

Comment:
@imuchnik might know how this one snuck in here -- was it an unused import? If so I will break it out into its own commit.